.github/workflows/sast-linters.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         with:
           version: v1.54
           skip-cache: true
-          args: --timeout=20m
+          args: --timeout=20m --out-format=json
```

Comment:
The time out here is longer than the timeout for the job on Line 57. I think we should use the timeout at the job level only and remove it from here.